### PR TITLE
Add new schemas and their attributes.

### PR DESCRIPTION
**Pull request by faustow**

### CODE REVIEW

File: dock-schemas.rst [hunk 1]
```diff
@@ -0,0 +1,148 @@
+Available schemas
+==================================
+
+Currently, the following schemas are available to use in the Dock protocol:
```
Comment (mclate) [inline dock-schemas.rst]: I'd rephrase it as such: "Currently dock  provides following schemas as a reference examples"

File: dock-schemas.rst [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+- contacts
+- education
+- email
+- empty
+- jobExperience
+- jobRecommendations
+- skills
+- userProfile
+
+A more detailed description of each schema can be found below.
```
Comment (mclate) [inline dock-schemas.rst:17]: Do we need this descriptions? They don't provide any useful info other than links to schemas. Fields one can see in the schema itself  (with way more info than in this list)

File: dock-schemas.rst [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  - startMonth
+  - startYear
+  - endMonth
+  - endYear
+  - isCurrent
+
+jobRecommendations
+------------------
+
+- Url: https://github.com/getdock/schemas/jobRecommendations.json
```
Comment (mclate) [inline dock-schemas.rst]: This schema gives no correlation to what job is being recommended.

Comment (faustow) [reply dock-schemas.rst]: There is no such information, recommendations and jobs are not related.

File: dock-schemas.rst (hunk 2 above)
Comment (faustow) [reply dock-schemas.rst:17]: Unsure if we really need them. They do constitute a quick high level overview of what's possible.

File: dock-schemas.rst (hunk 1 above)
Comment (faustow) [reply dock-schemas.rst]: Done

File: dock-schemas.rst (hunk 2 above)
Comment (mclate) [reply dock-schemas.rst:17]: Issue with having them here is that now we have to support two different places where we define schemas and fields, while the purpose of schemas (and schemas repo) is to be self-explanatory.

**Review by pswies — APPROVED** [review]

File: dock-schemas.rst (hunk 2 above)
Comment (foravneet) [reply dock-schemas.rst:17]: High level list is useful for biz & product, I agree that it will be a maintenance overhead for us though. I would prefer to still keep it but if you think it's easy to miss keeping this is sync then lets delete it. Its more important to have a 'correct' source that everyone refers.
